### PR TITLE
Add example usage

### PR DESCRIPTION
**Pull request by emilroz**
Add example usage in Bash and Python.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,45 @@ The token format is as follows:
...
+    echo "Connecting to ${host}:${port} with key ${key}"
+else
+    echo "No valid token found"
+    exit -1
+fi
+```
+
+### Python:
+```python
+import subprocess
```
Comment (chris-allan) [inline README.md]: Maybe just use the Python API?

```
from omero_user_token import getter
token = getter()
if token is not None:
    omero_session_key = token[:token.find('@')]
    host, port = token[token.find('@') + 1:].split(':')
    ...
```

Or maybe we just add a `login()` that does this for you and either gives you a logged in client or `None`?

File: README.md [hunk 2]
```diff
@@ -32,6 +32,45 @@ The token format is as follows:
 
     <omero_session_key>@<host>:<port>
 
+## Example usage
+
+### Bash:
+```bash
+if omero_user_token get; then
```
Comment (chris-allan) [inline README.md]: Try this:

```
token=$(omero_user_token get)
if [ $? -ne 0]; then
    echo "No valid token found"
    exit 1
fi
key=$(echo "${token}" | sed -e 's/^\(.*\)@.*:.*$/\1/')
host=$(echo "${token}" | sed -e 's/^.*@\(.*\):.*$/\1/')
port=$(echo "${token}" | sed -e 's/^.*@.*:\(.*\)$/\1/')
echo "Connecting to ${host}:${port} with key ${key}"

**Review by chris-allan — COMMENTED** [review]
Added some thoughts.

File: README.md (hunk 1 above)
Comment (emilroz) [reply README.md]: The problem with `token = getter()` is that it uses sys.exit() for all the token invalid conditions and ends with a spectacular stack trace. No ideal at the moment.

`login()` approach seems to me a better way to handle it.

Comment (chris-allan) [reply README.md]: Agreed. Do you want to try and refactor it here or want me to do it? It's not a big change.

Comment (emilroz) [reply README.md]: Docs updated.

File: README.md (hunk 2 above)
Comment (emilroz) [reply README.md]: Docs updated

**Comment by emilroz** [comment]
Closing in favor of https://github.com/glencoesoftware/omero-user-token/pull/5

**Pull request closed without merging**